recipes/georaster/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "georaster" %}
+{% set version = "1.15" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  md5: 72f2e8c57aee8cb01b689025f75a0d90
+
+build:
+    number: 0
+    script: python setup.py install
+
+requirements:
+  build:
+    - python
+    - numpy
```

Comment:
It does not look like `numpy` ,`scipy`, `gdal` and `pyproj` are required to build the recipe.